apps/api/src/server/repositories/document-metadata.repository.js

```diff
@@ -5,6 +5,16 @@ import { databaseConnector } from '../utils/database-connector.js';
  * @returns {import('@prisma/client').PrismaPromise<import('@pins/api').Schema.DocumentVersion>}
  */
 export const upsert = (metadata) => {
+	if (metadata.datePublished) {
+		metadata.publishedDate = metadata.datePublished;
```

Comment:
@LetsMakeItTechnical  I removed the new Date() from here, I'm sending it in a date format from the frontend.
this is just changing the names of the fields for the db upsert-ing